public/app/listeners/signOut.js

```diff
@@ -0,0 +1,26 @@
+const { SIGN_OUT_CHANNEL } = require('../config/channels');
+const { ERROR_GENERAL } = require('../config/errors');
+const logger = require('../logger');
+
+const signOut = (mainWindow, db) => async () => {
+  try {
+    db.set('user.authenticated', false).write();
+
+    // update user in users collection
+    const user = db.get('user').value();
+    db.get('users')
+      .find({ userId: user.userId })
+      .assign(user)
+      .write();
+
```

Comment:
Maybe a `// todo` for here would be to save the length of the session in an analytics collection (for general desktop based analytics).